third_party/build/build.py

```diff
@@ -76,8 +77,8 @@ def strsplitbytes(the_bytes):
     return [item for item in re.split(r'\s+', decoded) if item]
 
 
-def buildscript(package_name):
-    return './bin/' + package_name + '_build.sh'
+def buildscript(package_name: str) -> str:
+    return glob.glob(f'./bin/{package_name}*.sh')[0]
```

Comment:
Is there a reason we want to support anything other than underscore? Could we just have the underscore as the only naming scheme?

Reply:
ah, I see - this explains my comments above. I would suggest to rename the files and keep the original naming scheme.

Reply:
I don't have a strong opinion on this, tbh. Leaving the filenames as they are seemed the least intrusive way but renaming them is totally fine with me.